### DISCUSSION TITLE
schema的default属性无效

### DISCUSSION THREAD

**Discussion by caixianglin** [alibaba/formily#2703] [Q&A 2.x]
![image](https://user-images.githubusercontent.com/10279815/147735020-2fa25a9e-2c61-4437-9d82-5facc2a099c6.png)
![image](https://user-images.githubusercontent.com/10279815/147735173-1892e4cf-ff71-4110-b28a-293883c43798.png)

**Comment by caixianglin** [comment]
环境：自增卡片ArrayCollapse里面

**Reply by janryWang** [reply]
给复现
